src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -105,6 +110,8 @@ object Import {
   }
 
   val Bundle = config("bundle") extend Universal
+
+  val Configuration = config("config")
```

Comment:
Sure. I took it out because I noticed it was doing a lot of unnecessary stuff, ie. creating POMs etc.  ?